functions/getLocalLegislatorsData/findLocalLegislators.js

```diff
@@ -1,56 +1,35 @@
 const axios = require('axios')
-const functions = require('firebase-functions')
+const path = require('path');
+const https = require('https');
+const rootCas = require('ssl-root-cas').inject();
 
-const query = `
-query getLocalLegislators($latitude: Float, $longitude: Float) {
-  senator: people(latitude: $latitude, longitude: $longitude, memberOf: "ocd-organization/1a75ab3a-669b-43fe-ac8d-31a2d6923d9a", first: 1) {
-    edges {
-      node {
-        id
-        name
-      }
-    }
-  }
-  representative: people(latitude: $latitude, longitude: $longitude, memberOf: "ocd-organization/ca38ad9c-c3d5-4c4f-bc2f-d885218ed802", first: 1) {
-    edges {
-      node {
-        id
-        name
-      }
-    }
-  }
-}
-`
+rootCas.addFile(path.resolve(__dirname, 'certs/intermediate.pem'));
+const httpsAgent = new https.Agent({ca: rootCas});
 
 function findLocalLegislators(coordinates) {
-  return axios
-    .post(
-      'https://openstates.org/graphql',
-      {
-        query,
-        variables: {
-          latitude: coordinates.lat,
-          longitude: coordinates.lng,
-        },
-      },
-      {
-        headers: {
-          'X-API-KEY': functions.config().keys.openstates_api_key,
-        },
+  return axios.get('https://malegislature.gov/Legislators/GetDistrictByLatLong', {
+    params: {
+      latitude: coordinates.lat,
+      longitude: coordinates.lng,
+      isDistrictSearch: false,
+    },
+      httpsAgent,
+  }).then(response => {
+    memberCodes = {}
+    response.data.districts.map(({ Branch, UserMemberCode }) => {
+      if (Branch == 'Senate') {
```

Comment:
Yeah, the MA legislature endpoint response has them capitalized like that. I figured it's not worth renaming the variables since it's just a few lines of code.